src/prototypes/draft-email-textarea/demo/demo.scss

```diff
@@ -0,0 +1,131 @@
+@use '../../../compiled/tokens/scss/size';
+@use '../../../compiled/tokens/scss/color';
+@use '../../../compiled/tokens/scss/ease';
+@use '../../../compiled/tokens/scss/z-index';
+@use 'sass:color' as sass-color;
+@use '../../../mixins/ms';
+
+///
+/// Tooltip styles
+///
+/// TODO Consider making a Tooltip patterns component in the future
+///
+
+$tooltip-color-background: var(--theme-color-background-secondary);
+$tooltip-color-border: #{sass-color.change(
+    color.$brand-primary-darker,
+    $alpha: 0.4
+  )};
+
+.c-tooltip {
+  border: ms.step(-4) solid $tooltip-color-background;
+  border-radius: size.$border-radius-medium;
+  background: $tooltip-color-background;
+  color: var(--theme-color-text-emphasis);
+  box-shadow: 0 0 0 size.$edge-medium $tooltip-color-border,
+    size.$edge-large size.$edge-large 0 size.$edge-medium #{sass-color.change(
+        color.$brand-primary-darker,
+        $alpha: 0.2
+      )};
+  inline-size: max-content; // Is this the right way to handle this?
```

Comment:
I'm unsure if this is the best way to handle the following bug. Notice the tooltip text breaks into two lines (it should be one line):

<img width="423" alt="Screen Shot 2023-01-09 at 5 17 53 AM" src="https://user-images.githubusercontent.com/459757/211317039-1a173fb5-9c1b-4637-be57-2443627e1cd0.png">


It seems to be working. 🤷🏽

Reply:
If your goal is to prevent the text from breaking, would `white-space: nowrap` be more explanatory?